src/marlin/mod.rs

```diff
@@ -64,16 +63,9 @@ pub fn generate_proof(
     constraint_system: ConstraintSystemRef,
     proving_key: ProvingKey,
     rng: &mut StdRng,
-) -> Result<Vec<u8>> {
-    let proof = MarlinInst::prove_from_constraint_system(&proving_key, constraint_system, rng)
-        .map_err(|_e| anyhow!("Error in prove_from_constraint_system"))?;
-
-    let mut bytes_proof = Vec::new();
-    proof
-        .serialize(&mut bytes_proof)
-        .map_err(|_e| anyhow!("Error serializing proof"))?;
-
```

Comment:
I would suggest to add a new method to have serialization and deserialization

Reply:
Done!

Reply:
Great